manifests/tftp/copy_file.pp

```diff
@@ -0,0 +1,11 @@
+# Copy a TFTP file
+define foreman_proxy::tftp::copy_file(
+  $target_path,
+  $source_file = $title,
+) {
+  $filename = inline_template('<%= File.basename(source_file) %>')
```

Comment:
But not in the latest release (4.5.0), as far as I could see yesterday, only in current master.